src/markdown-pages/install-with-kurl/cis-compliance.md

```diff
@@ -0,0 +1,48 @@
+---
+path: "/docs/install-with-kurl/cis-compliance"
+date: "2021-12-14"
+weight: 24
+linktitle: "CIS Compliance"
+title: "CIS Compliance"
+isAlpha: false
+---
+The kURL installer can be configured to be Center for Internet Security (CIS) compliant. This is an opt-in feature that is configured in the kURL specification by setting the `kurl.cisCompliance` field to `true`.
+
+For information about known limitations, see [Known Limitations](#known-limitations).
+
+For more information about CIS security compliance for Kubernetes, see the [CIS benchmark information](https://www.cisecurity.org/benchmark/kubernetes).
```

Comment:
Is there a reason why we don't open the link in a new tab since it is external to our docs? so something like this

<a href="https://www.cisecurity.org/benchmark/kubernetes" target="_blank">CIS benchmark information</a>.

Reply:
Apparently opening links in new tabs isn't supported in classic Markdown. You have to Kramdown to do it, and I'm not sure if that would break our Netlify previews. I checked some other external links in this repository and they do not open in new tabs either.